lambdas/build-user-identity/src/main/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandler.java

```diff
@@ -138,6 +146,15 @@ public APIGatewayProxyResponseEvent handleRequest(
                             ipvSessionItem.getVot(),
                             ipvSessionItem.getCurrentVcStatuses());
 
+            if (configService.enabled(CoreFeatureFlag.USE_CONTRA_INDICATOR_VC)) {
+                String signedCiMitJwt =
+                        ciMitService.getContraIndicatorsVcAsJwtString(
+                                userId,
+                                clientOAuthSessionItem.getGovukSigninJourneyId(),
+                                ipAddress);
```

Comment:
Looks like `ipAddress` is not available for `BuildUserIdentity` - see commit `f1426ef6`. To be consistent with the audit call we should pass null

Reply:
Thanks catch this point. Done.